var/da/da_radiance/mod_qc.f90

```diff
@@ -0,0 +1,338 @@
+  module mod_qc
+!   use netcdf
```

Comment:
I would use a more relevant name like mod_clddet_ahi or sth more general mod_clddet_geoir if it is applicable to both AHI and ABI.